pepper-apis/testutil-pom.xml

```diff
@@ -1,89 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-
-    <parent>
-        <groupId>org.broadinstitute.ddp</groupId>
-        <artifactId>pepper-base</artifactId>
-        <version>SNAPSHOT</version>
-        <relativePath>parent-pom.xml</relativePath>
-    </parent>
-
-    <groupId>org.broadinstitute.ddp</groupId>
-    <artifactId>pepper-testutil</artifactId>
```

Comment:
I don't think anyone uses this. Couldn't figure out a way to add this to `scripts/ddp` so just removing it. You can probably run it via Intellij + JUnit setup.